save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/utils/FileUtils.kt

```diff
@@ -0,0 +1,24 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package org.cqfn.save.utils
+
+import java.io.File
+import java.io.FileNotFoundException
+import java.nio.file.Files
+import java.nio.file.StandardCopyOption
+
+/**
+ * Move [source] into [destinationDir], while also copying original file attributes
+ *
+ * @param source source file
+ * @param destinationDir destination directory
+ * @throws FileNotFoundException if source doesn't exists
+ */
+fun moveFileWithAttributes(source: File, destinationDir: File) {
+    if (!source.exists()) {
+        throw FileNotFoundException("Source file $source doesn't exist!")
+    }
+
+    Files.copy(source.toPath(), destinationDir.resolve(source.name).toPath(), StandardCopyOption.REPLACE_EXISTING, StandardCopyOption.COPY_ATTRIBUTES)
+    Files.delete(source.toPath())
```

Comment:
`Files.move`?

Reply:
Unfortunately, `Files.move` doesn't support `StandardCopyOption.COPY_ATTRIBUTE` option, so I used this workaround method

Reply:
Hm, that's strange. They even have it described in doc: `An implementation may also attempt to copy other file attributes but is not required to fail if the file attributes cannot be copied.`

Reply:
Besides that I met exceptions `unsupported option`, here it's also can be seen https://docs.oracle.com/javase/tutorial/essential/io/move.html, that they support only `REPLACE_EXISTING` and `ATOMIC_MOVE` options